src/entities/StoreUser.ts

```diff
@@ -19,7 +19,7 @@ export default class StoreUser {
       case n.length > 12:
         throw Error("El Nombre debe poseer como máximo 12 caracteres.");
       case !/^[a-zA-Z]+$/.test(n):
-        throw Error("El Nombre debe contener solo letras.");
+        throw Error("El Nombre debe contener solo constras.");
```

Comment:
```suggestion
        throw Error("El Nombre debe contener solo letras.");
```

fijate que hiciste un replace all y se te cambiaron algunas palabras!!